guides/app-guides/impersonator.md

```diff
@@ -0,0 +1,28 @@
+---
+description: How to impersonate an address on app.angle.money
```

Comment:
I'd make this guide directly in the main App Guides page and in fact do a more detailed guide for settings